guides/common/modules/proc_viewing-the-project-ansible-modules.adoc

```diff
@@ -1,39 +1,13 @@
 [id="Viewing_the_Ansible_Modules_{context}"]
 = Viewing the {Project} Ansible Modules
 
-ifdef::satellite,orcharhino[]
-You can view the installed {Project} Ansible modules by listing the content of the following directory:
-
-[options="nowrap" subs="+quotes,attributes"]
-----
-# ls {ansiblefilepath}
-----
-
-[NOTE]
-====
-At the time of writing, the `ansible-doc -l` command does not list collections yet.
-====
-
-endif::[]
-
-ifndef::satellite,orcharhino[]
-
-Starting with Ansible 2.10, you can view the installed {Project} Ansible modules by running:
+You can view the installed {Project} Ansible modules by running:
 
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # ansible-doc -l {ansible-namespace}
 ----
 
-When using Ansible before 2.10, you can view the installed {Project} Ansible modules by listing the content of the following directory:
-
-[options="nowrap" subs="+quotes,attributes"]
-----
-# ls {ansiblefilepath}
-----
-
-endif::[]
-
 ifndef::orcharhino[]
```

Comment:
I'm curious why this is `ifndef orcharino`

Reply:
ATIX does not run a Ansible Galanxy for orcharhino. This would link to "https://galaxy.ansible.com/theforeman/foreman" for orcharhino.

IMHO this is OK as is.